Documentation/devel/update-code-for-all-services.sh

```diff
@@ -15,12 +16,13 @@ SERVICES=(#bridge
           pbforwarder
           router
           simplebridge
-          simpleforwarder)
+          simpleforwarder
+          transparent-helloworld)
 
```

Comment:
Should we exit this script if no `POLYCUBE_HOME` is set?

```suggestion

if [ -z "$POLYCUBE_HOME" ]; then
    echo "POLYCUBE_HOME is not set"
    exit 1
fi
```

Reply:
I added it, thanks!